### PR TITLE
add coolix receiver

### PR DESCRIPTION
**Pull request by glmnet**
## Description:
Coolix receiver :)

**Related issue (if applicable):** fixes <link to issue>

**Pull request in [esphome](https://github.com/esphome/esphome) with YAML changes (if applicable):** esphome/esphome# https://github.com/esphome/esphome/pull/645

## Checklist:

  - [x] Branch: `next` is for changes and new documentation that will go public with the next ESPHome release. Fixes, changes and adjustments for the current release should be created against `current`.

### CODE REVIEW

File: components/climate/coolix.rst [hunk 1]
```diff
@@ -36,13 +39,23 @@ Configuration variables:
...
 
 - **id** (*Optional*, :ref:`config-id`): Manually specify the ID used for code generation.
 - **transmitter_id** (*Optional*, :ref:`config-id`): Manually specify the ID of the remote transmitter.
 
+.. _coolix-receiver_id:
+
+Using a receiver
+----------------
+
+When using a receiver it is recommended to put the IR receiver as close as the equipment's IR receiver
```
Comment (OttoWinter) [inline components/climate/coolix.rst]:
```suggestion
When using a receiver it is recommended to put the IR receiver as close as possible to the equipment's IR transmitter
```

File: components/climate/coolix.rst [hunk 2]
```diff
@@ -36,13 +39,23 @@ Configuration variables:
...
+so the chances of receiving (and not receiving) IR signals are the same as of the AC unit.
```
Comment (OttoWinter) [inline components/climate/coolix.rst]: I mean yes, that's one way of putting it, but wouldn't it be simpler to say: Just "put the receiver as close as possible"? Is there any reason the chance of receiving and transmitting have to be equal?

File: components/climate/coolix.rst [hunk 3]
```diff
@@ -36,13 +39,23 @@ Configuration variables:
...
+It is recommended to use a high tolerance, like 50 or so in the receiver configuration.
```
Comment (OttoWinter) [inline components/climate/coolix.rst]: A short example configuration below would really help :)

File: components/climate/coolix.rst (hunk 2 above)
Comment (glmnet) [reply components/climate/coolix.rst]: Ohhh what I'm trying to say is to put the receiver as close as possible to the equipment's original IR receiver, so both receivers next to each other.
This way when a signal is sent by someone using the remote control unit, it really does not depend much where this person is on the room.

**Pull request merged**